static/js/speech.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Speech Recognition Helper.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+'use strict';
+
+// eslint-disable-next-line no-unused-vars
+var Speech = {
+
+    /**
+    * Initialise menu.
+    */
+    init: function(that) {
```

Comment:
nit: `that` should be called `app` or something that makes it clear what's happening

Reply:
Fixed!